webclient/webclient/src/main/resources/META-INF/native-image/reflection-config.json

```diff
@@ -0,0 +1,8 @@
+[
```

Comment:
Rename to `reflec-config.json` and it would be picked up automatically.
Also move this file to
`META-INF/native-image/io.helidon.webclient/helidon-webclient` - that is the recommended location (groupId/artifactId)